src/create/note.ts

```diff
@@ -0,0 +1,71 @@
+import * as color from 'picocolors';
```

Comment:
Improving visibility of message in `note` from `@clack/prompts`.

`@clack/prompts` dims the `note` message color making it hard to see.